### PR TITLE
fix: add docs panel

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #9056

### Description:

Screenshot from 2024-02-21 17-19-40
### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/features/scheduler/components/SchedulerModal.tsx [hunk 1]
```diff
@@ -34,11 +35,21 @@ const SchedulersModal: FC<
                     <Group spacing="xs">
                         <MantineIcon icon={IconBell} size="lg" color="gray.7" />
                         <Text fw={600}>Alerts</Text>
+                        <DocumentationHelpButton
+                            href="https://docs.lightdash.com/guides/how-to-create-alerts"
```
Comment (rephus) [inline packages/frontend/src/features/scheduler/components/SchedulerModal.tsx:39]: depends on https://github.com/lightdash/lightdash/pull/9022

File: packages/frontend/src/features/scheduler/components/SchedulerModal.tsx [hunk 2]
```diff
@@ -34,11 +35,21 @@ const SchedulersModal: FC<
...
+                            pos="relative"
+                            top="2px"
+                        />
                     </Group>
                 ) : (
                     <Group spacing="xs">
                         <MantineIcon icon={IconSend} size="lg" color="gray.7" />
                         <Text fw={600}>Scheduled deliveries</Text>
+                        <DocumentationHelpButton
+                            href="https://docs.lightdash.com/guides/how-to-create-scheduled-deliveries"
```
Comment (magnew) [inline packages/frontend/src/features/scheduler/components/SchedulerModal.tsx:49]: Do we think these should open in another window? These only show up in part of the app (this modal) where there is an incomplete form and no saved draft. So if they did already do something in this form, then click that link, they will always loose state. 

@PriPatel any thoughts?

Comment (PriPatel) [reply packages/frontend/src/features/scheduler/components/SchedulerModal.tsx:49]: Oh woops! yes these should definitely open in a new tab @rephus @magnew

Comment (rephus) [reply packages/frontend/src/features/scheduler/components/SchedulerModal.tsx:49]: this component already opens links in a new tab, we use this component in a few other places . Does it not open in a new tab for you ?

Comment (magnew) [reply packages/frontend/src/features/scheduler/components/SchedulerModal.tsx:49]: Re-tested and this does what I expect. I must have been losing it yesterday. Sorry about the confusion!

**Review by magnew — APPROVED** [review]

**Pull request merged**